2996_reflection/reflection.md

```diff
@@ -2487,21 +2557,16 @@ Add a new subsection of [expr.unary]{.sref} following [expr.delete]{.sref}
 ::: addu
 **The Reflection Operator   [expr.reflect]**
 
-[#]{.pnum} The unary `^` operator (called _the reflection operator_) produces a prvalue --- called _reflection_ --- whose type is the reflection type (i.e., `std::meta::info`).
+[#]{.pnum} The unary `^` operator (called _the reflection operator_) produces a prvalue --- called a _reflection_ --- whose type is the reflection type (i.e., `std::meta::info`).
 That reflection represents its operand.
 
 [#]{.pnum} Every value of type `std::meta::info` is either a reflection of some operand or a *null reflection value*.
 
-[#]{.pnum} An ambiguity can arise between the interpretation of the operand of the reflection operator as a `$type-id$` or a `$cast-expression$`; in such cases, the `$type-id$` treatment is chosen.
-Parentheses can be introduced to force the `$cast-expression$` interpretation.
-
-
 [#]{.pnum}
 
 ::: example
 ```
-static_assert(is_type(^int()));    // ^ applies to the type-id "int()"; not the cast "int()"
-static_assert(!is_type(^(int()))); // ^ applies to the the cast-expression "(int())"
+static_assert(is_type(^int()));    // ^ applies to the type-id "int()"
```

Comment:
This example is kind of hanging now, since it used to describe the ambiguity. But still useful to have... somewhere. I guess worry about it later.